Public/Sdk/SelfHost/BuildXL/Testing/QTest/qtestFramework.dsc

```diff
@@ -62,6 +62,17 @@ function runMultipleQTests(args: TestRunArguments) : File[]
     ];
 }
 
+function getLogDirectory(args:TestRunArguments, pipType: string) : Directory
+{
+    const baseDir = Environment.hasVariable("BUILDXL_IS_IN_CLOUDBUILD")

```

Comment:
>BUILDXL_IS_IN_CLOUDBUILD [](start = 45, length = 24)

Adding Iman since I beleive he was trying to keep the graph between IC's and cloudbuild consistent

Reply:
@anirudhsanthiar  is just refactoring the existing code into a function. Let's treat the existing IN_CLOUDBUILD pattern as a separate problem and not address in the PR